### PR TITLE
Fixes DOC-1311

### PR DESCRIPTION
**Pull request by lamagnifica**
@martynjames, @marjev, @mulby, @stroilova please check for accuracy and
completeness
@mhoeber, please copy edit/proofread
Needed for week of 25 Feb (?)

### CODE REVIEW

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 1]
```diff
@@ -2596,6 +2599,53 @@ the response.
...
+``event`` **Member Fields**:
+
+.. list-table::
+   :widths: 15 15 60
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Details
+   * - ``displayedin``
```
Comment (mulby) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: I believe they later changed this to "displayed_in"

**Comment by mulby** [comment]
1 small nit, then :+1:

**Comment by martynjames** [comment]
lgtm :+1:

**Comment by marjev** [comment]
:+1:

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 2]
```diff
@@ -2642,6 +2645,52 @@ the response.
...
+``event`` **Member Fields**:
+
+.. list-table::
+   :widths: 15 15 60
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Details
+   * - ``displayed_in``
```
Comment (stroilova) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:2927]: Hi @lamagnifica  I'm seeing this field called "displayedin" with no underscore in the sample events. Asked in a question on the Google Drive PR: https://github.com/edx-solutions/xblock-google-drive/pull/13

I think displayed_in was the most recently agreed-upon name, but just want to double check what events are being emitted.

Comment (stroilova) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:2927]: Update: we validated on Stage that displayed_in is the right name.

**Pull request merged**